User Registration

angel23v opened the pull request:
Use devise gem to authenticate users, custom views generated & modify some behaviors on navbar

For more info see: https://github.com/BrightCoders-Institute/s5a1-administrador-de-eventos-team4/issues/22

angel23v committed:
User Registration & Login

Use devise gem to authenticate users, custom views generated & modify some behaviors on navbar

Co-authored-by: Bryanroal<bryanroal27@gmail.com>

MauricioNC commented on app/views/devise/confirmations/_navbar.hml.erb:
```diff
@@ -0,0 +1,21 @@
+          <%= link_to "Iniciar", events_index_path , class: "btn btn-dark text-light"%>
+        </li>
+      </ul>
+      <form class="d-flex" role="search">
+        <input class="form-control me-2" type="search" placeholder="Buscar Evento" aria-label="Search">
+        <button class="btn btn-outline-success" type="submit">Buscar</button>
+      </form>
```
Just as suggestion, you can try in another PR create the search functionality with Rails

angel23v replied:
In fact, we chose to eliminate that option 'cause we saw that we will later use a filter. 
We forgot to delete it in some files but surely we will create another PR.
Thank you.

MauricioNC commented on app/views/devise/sessions/_navbar.html.erb:
```diff
@@ -0,0 +1,29 @@
+    </button>
+    <div class="collapse navbar-collapse" id="navbarSupportedContent">
+      <ul class="navbar-nav me-auto mb-2 mb-lg-0">
+        <li class="nav-item">
+          <% if signed_in? && current_page?(events_main_path)%>
+            <%= link_to "Iniciar", events_index_path , class: "btn btn-dark text-light"%>
+          <% end %>
+        </li>
```
What happens if the user is not signed in or the current page is not equals to events_main_path?.
If the condition is not met do you'll leave an empty li tag?

Same for the other li tag and for the app/views/events/_navbar.html.erb file

juperameza reviewed (changes requested):
Remove the navbar file and just add it in the layout directory

juperameza commented:
> Remove the navbar file and just add it in the layout directory

And do everything in a single language

angel23v committed:
Attend issues

Remove the navbar file, fix everything in a single language & fix action on navbar

Co-authored-by: Bryanroal<bryanroal27@gmail.com>
Co-authored-by: SharonBel <belmontsharon77@gmail.com>

juperameza commented on config/routes.rb:
```diff
@@ -17,6 +18,70 @@
   post '/events', to: 'events#create'
 
+  devise_scope :user do
+    get '/users/log_in' => 'devise/sessions#new'
+  end
+
+  devise_scope :user do
+    get '/users/sig_in' => 'devise/sessions#new'
+  end
+
+  devise_scope :user do
+    post '/users/sign_in' => 'devise/sessions#create'
+  end
+
+  devise_scope :user do
+    get '/users/sign_out' => 'devise/sessions#destroy'
+  end
```
Why are you doing this

juperameza replied:
and in multiple lines too

angel23v committed:
Clear routes
